critical/lambda_logging.tf

```diff
@@ -1,22 +1,14 @@
-module "elastic_log_forwarder" {
-  source = "./modules/elastic_log_forwarder"
-
-  kinesis_log_stream_arn = module.kinesis_log_destination.kinesis_stream.arn
-  es_data_stream         = module.esf_data_stream.name
-  es_api_key_secret      = "elasticsearch/logging/esf/api_key"
-}
-
 module "kinesis_log_destination" {
   source = "./modules/kinesis_log_destination"
 
-  name               = "esf-logs"
+  name               = "elasticsearch-forwarder-logs"
   source_account_ids = values(local.account_ids)
 }
 
 // Put this in parameter store rather than as an output
 // so that our Lambdas don't need access to the whole critical stack state
 locals {
-  esf_destination_parameter_name = "/logging/esf/destination_arn"
+  esf_destination_parameter_name = "/logging/forwarder/destination_arn"
```

Comment:
nit: worth removing the 'esf' name here?

Reply:
oops, yes